src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx

```diff
@@ -21,314 +21,230 @@ After setting up our SonarQube integration, we give you a dashboard for your Son
-      CONNECTION_URL: service:jmx:rmi://<YOUR_IP>:9011/jndi/rmi://<YOUR_IP>:9011/jmxrmi
-      REMOTE_MONITORING: "true"
-    interval: 15s
-    labels:
-      env: staging
-```
-</Collapser>
-</CollapserGroup>
-
+        
+    </Step>
+    <Step>
+## Configure NRI-Prometheus      
+
+1. Run the following command to create a NRI-Prometheus config file:
+
+  ```shell
+  touch /etc/newrelic-infra/integrations.d/nri-prometheus-config.yml
+  ```
+2. Paste the following snippet into the new config file. Be sure to update the `cluster_name` and `urls` fields:
+
+  ```yml
+  integrations:
+    - name: nri-prometheus
+      config:
+        # When standalone is set to false nri-prometheus requires an infrastructure agent to work and send data. Defaults to true 
+        standalone: false
+
+        # When running with infrastructure agent emitters will have to include infra-sdk 
+        emitters: infra-sdk
+
+        # The name of your cluster. It's important to match other New Relic products to relate the data.
+        cluster_name: "YOUR_DESIRED_CLUSTER_NAME"
+
+        targets:
+          - description: Sonarqube metrics list
+            urls: ["http://user_name:password@YOUR_HOST_IP:9000/api/monitoring/metrics"]
+
+        #     tls_config:
+        #     ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+        #     cert_file_path: "/etc/etcd/etcd-client.crt"
+        #      key_file_path: "/etc/etcd/etcd-client.key"
+
+        # Whether the integration should run in verbose mode or not. Defaults to false
+        verbose: false
+
+        # Whether the integration should run in audit mode or not. Defaults to false.
+        # Audit mode logs the uncompressed data sent to New Relic. Use this to log all data sent.
+        # It does not include verbose mode. This can lead to a high log volume, use with care
+        audit: false
+
+        # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
+        # scrape_timeout: "30s"
+
+        # Length in time to distribute the scraping from the endpoints
+        scrape_duration: "5s"
+
+        # Number of worker threads used for scraping targets.
+        # For large clusters with many (&gt;400) endpoints, slowly increase until scrape
+        # time falls between the desired `scrape_duration`.
+        # Increasing this value too much will result in huge memory consumption if too
+        # many metrics are being scraped.
+        # Default: 4
+        # worker_threads: 4
+
+        # Whether the integration should skip TLS verification or not. Defaults to false
+        insecure_skip_verify: true
+      timeout: 10s
+  ```
+    </Step>
+    <Step>
+## Install and configure the JMX integration
+
+To use the SonarQube integration, you need to also [install our JMX monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-install/). The JMX integration scrapes SonarQube data, which we will later turn into dashboards and queryable data.
```

Comment:
this is good context and i think a small sentence for nri-prometh + prostgresql steps that parallel this intro might be helpful. 

i'd approach these sentences as a "mini narrative" that threads these separate install steps together and explains how the data from these diff install processes "come together" to tell a SQ story